<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#loginEdit {
border: none;
border-bottom: 2px solid rgb(3, 76, 46);
border-top: 2px solid rgb(3, 76, 46);
background-color: rgba(9, 9, 9, 0);
color: rgba(255, 255, 255, 200);
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passEdit {
border: none;
border-bottom: 2px solid rgb(3, 76, 46);
border-top: 2px solid rgb(3, 76, 46);
background-color: rgba(9, 9, 9, 0);
color: rgba(255, 255, 255, 200);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#check_btn {
color: rgb(255, 255, 255);
border: 2px Solid rgb(255, 255, 255);
background-color: none
}
QPushButton#check_btn:hover {
color: rgb(255, 255, 255);
border: 5px Solid rgb(255, 255, 255);
background-color: none;
}</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>QLineEdit#loginEdit {</normaloff>QLineEdit#loginEdit {</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="Error_lbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>650</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passRepeat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passRepeat {
border: none;
border-bottom: 2px solid rgb(3, 76, 46);
border-top: 2px solid rgb(3, 76, 46);
background-color: rgba(9, 9, 9, 0);
color: rgba(255, 255, 255, 200);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Повторить пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/mountains.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/exit_icon.jpg</normaloff>../../../../Desktop/exit_icon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>loginEdit</zorder>
  <zorder>passEdit</zorder>
  <zorder>check_btn</zorder>
  <zorder>Error_lbl</zorder>
  <zorder>passRepeat</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <resources>
  <include location="../../../../Downloads/background.qrc"/>
 </resources>
 <connections/>
</ui>
